test/application/inspectors/ensureNotRevokedByList.test.js

```diff
@@ -0,0 +1,22 @@
+import { ensureNotRevokedByList } from '../../../src/inspectors';
+import revokedAssertionsFixture from '../../application/domain/verifier/useCases/fixtures/revokedAssertionsFixture';
+
+describe('Inspectors test suite', function () {
+  describe('ensureNotRevokedByList method', function () {
+    describe('given it is called without revokedAssertions', function () {
+      it('should return nothing', function () {
+        expect(ensureNotRevokedByList()).toBe();
```

Comment:
`toBe(undefined)`?

Reply:
Good catch. This file is actually going away in the next PR, but the replacing part has the same test case. I'll update.